ppx/html.ml

```diff
@@ -0,0 +1,584 @@
+type attributeType = String | Int | Float | Bool | Style | Ref | InnerHtml
```

Comment:
I moved out of dom.mli since those get exposed on purpose to the user, while the html.ml/i shouldn't be. If makes more sense to keep it inside Dom, we can make a module called... Private__ or somekind to express it better. WDYT?

Reply:
Ah right, I didn't realize the `html.ml` file was in `ppx` folder. 👍 

Maybe just inline this inside this module in `ppx.ml` then? So it's clear that is something internal to the ppx.

About the modlue naming, I would use `module Dom` for consistency with the other one, and with [React.js docs](https://reactjs.org/docs/dom-elements.html). https://reactjs.org/docs/dom-elements.html, as HTML has different attributes than the props passed to these elements.

But these are all nits :)